spot-client/src/spot-tv/ui/components/join-info/join-info.js

```diff
@@ -56,21 +56,11 @@ class JoinInfo extends React.Component {
 
         return (
             <>
-                { `${shareDomain || windowHandler.getHost()}/` }
+                { `${shareDomain || windowHandler.getBaseUrl()}/` }
```

Comment:
Probably at some point the same change will have to be made for shareDomain somehow, but not today.